getting_started_moa/creating-first-moa-cluster.adoc

```diff
@@ -18,7 +18,7 @@ $ moactl init
 $ moactl create cluster --name <my-cluster-name>
 
 ## Watch your logs as your cluster creates
-$ moactl logs cluster <my-cluster-name> --watch
+$ moactl logs install <my-cluster-name> --watch
```

Comment:
Replaceables should [use underscores](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output).

Reply:
I'm gonna do a "fast-follow" PR and replace them all at once. I used "-" throughout.